spec/post_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  describe 'validations' do
+    it { is_expected.to validate_presence_of(:title) }
+
+    it 'should validate that comments_counter is greater than or equal to 0' do
+      post = Post.new(title: 'title', comments_counter: -1, likes_counter: 1)
+      expect(post).not_to be_valid
+      expect(post.errors[:comments_counter]).to include('must be greater than or equal 0')
+    end
+
+    it 'should validate that likes_counter is greater than or equal to 0' do
+      post = Post.new(title: 'title', comments_counter: 1, likes_counter: -1)
+      expect(post).not_to be_valid
+      expect(post.errors[:likes_counter]).to include('must be greater than or equal 0')
+    end
+
+    it 'title should not exceed 250 characters' do
+      mock_title = 'Schwebet die der nun in diesmal, trüben wiederholt mit schwankende lebt
+      folgenden herzen umwittert lied schatten. Schmerz getäuscht mein die seh
+      wirklichkeiten lauf wie der, versuch beifall nach das nach mir sich lied,
+      mich und sich es schwebet auf tage. Mit träne labyrinthisch hören
+      nebel vor. Und versuch äolsharfe umwittert meinem.
+      Sed ut tellus metus. Vivamus malesuada, purus sed venenatis rutrum, nunc ex
+      viverra elit,
+      ac bibendum orci turpis ut mi. Nulla facilisi. Fusce euismod, velit vel
+      ultrices ullamcorper,
+      nibh velit interdum leo, vel varius lacus sapien eu augue. Fusce euismod, velit
+      vel ultrices ullamcorper, nibh velit
+      interdum leo, vel varius lacus sapien eu augue. Sed eget nisl auctor, posuere enim eu,
+      eleifend nisl. Mauris bibendum volutpat nunc ac interdum. Donec euismod, quam at
+      elementum porttitor, odio neque consequat quam, nec lacinia nulla mi a ligula.
+      Sed sed risus eget sapien scelerisque vehicula. Sed faucibus turpis et ante
+      molestie, non gravida eros tincidunt. Nunc id urna diam. Fusce non est vel
+      erat convallis tincidunt. Nulla facilisi. Donec id volutpat nulla.'
+      post = Post.new(title: mock_title, comments_counter: 1, likes_counter: 1)
+      expect(post).not_to be_valid
+      expect(post.errors[:title]).to include('cannot exceed 250 characters')
+    end
```

Comment:
This is not the best way to write the test. In this case the test will always throw an error when the title length is less than 250.  In the image below I put in a title length of 50 and got an error.   It should rather be the other way round so that the test throws an error only when the title length is greater than 250. Kindly fix this. :pray: 

![Screenshot from 2023-09-30 19-06-07](https://github.com/yin-ka/blog-app/assets/65233860/7b805a54-eaf6-4939-835e-b326f489029e)